docs/rules/symbol-description.md

```diff
@@ -0,0 +1,61 @@
+# Require Symbol Description (symbol-description)
+
+The `Symbol` function may have optional description:
+
+```js
+var foo = Symbol("foo");
+
+var someString = 'bar';
+var bar = Symbol(someString);
+```
+
+
+Using `description` promotes easier debugging: when a symbol is logged the description is used:
+
+```js
+var foo = Symbol("foo");
+
+> console.log(foo);
+// Symbol(foo)
+```
```

Comment:
Could you provide an example of what would be printed if no description is provided? Just to explain to users who are not very familiar with `Symbol` why it's better to have description.